app/src/main/java/com/teamagam/gimelgimel/app/common/base/view/fragments/BaseDataFragment.java

```diff
@@ -36,6 +37,7 @@ public void onCreate(Bundle savedInstanceState) {
         mViewModel.init();
     }
 
+    @NonNull
```

Comment:
eh?
There's already @NotNull attribute on that method.
These annotation sucks anyways, they're useless.